recipe/meta.yaml

```diff
@@ -46,8 +46,9 @@ requirements:
         - g2clib
         - hdfeos2
         - hdfeos5
-        - libgcc
         - krb5
+        - libgcc  # [osx]
+        - libgfortran  # [osx]
```

Comment:
This will be included with `libgcc`. Also, not sure if this package exists for anything other that Linux.